.travis.yml

```diff
@@ -17,7 +17,7 @@ before_script:
 
 script:
   - if [ "$JOB" = "test" ]; then make -j test-ci; fi
-  - if [ "$JOB" = "lint" ]; then make -j code-quality-ci; fi
+  - if [ "$JOB" = "lint" ]; then make code-quality-ci; fi
```

Comment:
This was failing because `node_modules/*` was being deleted while the linting task was running, leading to ESLint not finding plugins.

My experience with `make` is limited, so maybe there's a better way to fix this? Maybe doing all the prep work ahead of time in its own `make` target and then running the checks in parallel after that?

Reply:
Okay, I think I fixed this by bootstrapping everything as a prerequisite and then running the linting jobs afterwards in parallel. One thing I'm not entirely sure of is if the sub `make` targets are also running in parallel.

This should be ready to go now!